contracts/NameWrapper.sol

```diff
@@ -280,6 +281,49 @@ contract NameWrapper is Ownable, ERC1155Fuse, INameWrapper {
         emit NameWrapped(ETH_NODE, label, wrappedOwner, _fuses);
     }
 
+    /**
+     * @dev Registers a new .eth second-level domain and wraps it.
+     *      Only callable by authorised controllers.
+     * @param label The label to register (Eg, 'foo' for 'foo.eth').
+     * @param wrappedOwner The owner of the wrapped name.
+     * @param duration The duration, in seconds, to register the name for.
+     * @param resolver The resolver address to set on the ENS registry (optional).
+     * @return expires The expiry date of the new name, in seconds since the Unix epoch.
+     */
+    function registerAndWrapETH2LD(
+        string calldata label,
+        address wrappedOwner,
+        uint duration,
+        address resolver,
+        uint96 _fuses
+    ) onlyController external override returns(uint expires) {
```

Comment:
Should we return the `node` and `labelhash` here just in case the next call needs to use it (in say a smart contract call) and they don't need to waste more gas to keccak the label or node again? Does returning extra things cost more gas?

Reply:
It does, and keccak is cheap, so it's probably not much of a saving to return it.